src/js/vic-v2/config/form.js

```diff
@@ -33,6 +35,8 @@ const {
   gender
 } = fullSchemaVIC.definitions;
 
+const twentyFiveMB = 26214400;
```

Comment:
Might be better as TWENTY_FIVE_MB, to follow our common constant pattern.

Reply:
Thanks @jbalboni for the helpful guidance (updated).